paper/paper.md

```diff
@@ -104,121 +106,142 @@ This study includes data from 11 participants (Mean age = 28.7, SD = 13.4, range
 ```r
 m_lm <- lm(mpg ~ qsec + wt, data = mtcars)
 report(m_lm)
-#> We fitted a linear model (estimated using OLS) to predict
-#> mpg with qsec and wt (formula: mpg ~ qsec + wt). The model
-#> explains a statistically significant and substantial
-#> proportion of variance (R2 = 0.83, F(2, 29) = 69.03, p <
-#> .001, adj. R2 = 0.81). The model's intercept, corresponding
-#> to qsec = 0 and wt = 0, is at 19.75 (95% CI [9.00, 30.49],
-#> t(29) = 3.76, p < .001). Within this model:
-#> 
-#>   - The effect of qsec is statistically significant and
-#> positive (beta = 0.93, 95% CI [0.39, 1.47], t(29) = 3.51, p
-#> = 0.001; Std. beta = 0.28, 95% CI [0.11, 0.44])
-#>   - The effect of wt is statistically significant and
-#> negative (beta = -5.05, 95% CI [-6.04, -4.06], t(29) =
-#> -10.43, p < .001; Std. beta = -0.82, 95% CI [-0.98, -0.66])
-#> 
-#> Standardized parameters were obtained by fitting the model
-#> on a standardized version of the dataset. 95% Confidence
-#> Intervals (CIs) and p-values were computed using a Wald
-#> t-distribution approximation.
+```
+
+::: {.blackbox data-latex=""}
+
+We fitted a linear model (estimated using OLS) to predict
+mpg with qsec and wt (formula: mpg ~ qsec + wt). The model
+explains a statistically significant and substantial
+proportion of variance (R2 = 0.83, F(2, 29) = 69.03, p <
+.001, adj. R2 = 0.81). The model's intercept, corresponding
+to qsec = 0 and wt = 0, is at 19.75 (95% CI [9.00, 30.49],
+t(29) = 3.76, p < .001). Within this model:
+
+  - The effect of qsec is statistically significant and
+positive (beta = 0.93, 95% CI [0.39, 1.47], t(29) = 3.51, p
+= 0.001; Std. beta = 0.28, 95% CI [0.11, 0.44])
+  - The effect of wt is statistically significant and
+negative (beta = -5.05, 95% CI [-6.04, -4.06], t(29) =
+-10.43, p < .001; Std. beta = -0.82, 95% CI [-0.98, -0.66])
+
+Standardized parameters were obtained by fitting the model
+on a standardized version of the dataset. 95% Confidence
+Intervals (CIs) and p-values were computed using a Wald
+t-distribution approximation.
+
+:::
```

Comment:
Doesn't this mean we will have to manually update this content if we decide to change anything about the reporting methods, and thereby lose any benefit offered by a dynamic document? 

I feel we are doing this formatting change prematurely. The right time to do it would be when we are ready for a submission. WDYT?

Reply:
Ah, so dumb. 

That's great then :)

Maybe we use a bit softer color, but should be good for now.